src/components/Tooltip/Tooltip.tsx

```diff
@@ -152,6 +120,23 @@ const Tooltip: React.FC<TooltipProps> = ({
     inverted = false
 }: PropsWithChildren<TooltipProps>) => {
     const [isVisible, setIsVisible] = React.useState(alwaysVisible);
+    const [triggerReference, setTriggerReference] = React.useState(undefined);
```

Comment:
Can you add some comments here explaining the purpose of the state variables and the mapping function at a high level? I don't assume people needing to know this unless they need to change something, but good to have

Reply:
It's a requirement for usePopper to work. I don't understand the concern. Can you explain better what you think is the issue?